docs/src/user/installing.md

```diff
@@ -12,30 +12,39 @@ rustup target add --toolchain stable wasm32-wasi
 
 ## Installing from Git
 
-Ambient can be installed through `cargo install`, which will download and build the repository. Our current minimum supported Rust version is <!-- rust-version-begin !-->1.67.0<!-- rust-version-end !-->, as we use recently-stabilised standard library features.
+*Ambient* can be installed through `cargo install` using git.
 
-To install the latest released version from Git, run the following:
+This will download the source and compile the runtime. Our minimum supported Rust version is <!-- rust-version-begin -->1.67.0<!-- rust-version-end -->.
+
+### Installing the latest published release
+
+This is recommended if you want a binary for your platform using the last stable release
 
 ```sh
-cargo install --git https://github.com/AmbientRun/Ambient.git --tag v0.2.1 ambient --locked
+cargo install --git https://github.com/AmbientRun/Ambient.git --tag v0.2.1 --locked --force ambient
 ```
 
-To install the latest version on the `main` branch, run the following:
+**Note**: If you are running a project outside of the `guest/rust` workspace, it is likely that the published version of the API will be incompatible with `main`, and you will need to specify the dependency manually.
+
+### Installing the latest development version
+
+This is recommended if you are developing and targeting the latest features of Ambient which have yet to release.
```

Comment:
> Ambient is actively developed on the `main` branch of the repository. This branch contains in-development changes, including new features, bugfixes and breaking changes. This method can be used if you would like to try out these changes.
>
> **Note**: The `main` branch is subject to frequent breaking changes, including potential new bugs and decreased stability, and is not a stable development target for projects. Use of the `main` branch is not recommended if you are unable to actively update your project to accommodate breaking changes.

Reply:
Sorry, but I'd prefer if you used this wording verbatim. It better conveys the trade-offs of using the development version.

Reply:
Is this better?